src/constants.ts

```diff
@@ -0,0 +1,165 @@
+export const CHAIN_IDs = {
+  MAINNET: 1,
+  OPTIMISM: 10,
+  ARBITRUM: 42161,
+  BOBA: 288,
+  POLYGON: 137,
+  GOERLI: 5,
+  MUMBAI: 80001,
+};
+
+// NOTE: All addresses should be checksummed
+export const TOKEN_SYMBOLS_MAP = {
+  ACX: {
+    name: "ACX",
+    symbol: "ACX",
+    decimals: 18,
+    addresses: {
+      [CHAIN_IDs.MAINNET]: "0x44108f0223A3C3028F5Fe7AEC7f9bb2E66beF82F",
+      [CHAIN_IDs.OPTIMISM]: "0xFf733b2A3557a7ed6697007ab5D11B79FdD1b76B",
+      [CHAIN_IDs.POLYGON]: "0xF328b73B6c685831F238c30a23Fc19140CB4D8FC",
+      [CHAIN_IDs.BOBA]: "0x96821b258955587069F680729cD77369C0892B40",
+      [CHAIN_IDs.ARBITRUM]: "0x53691596d1BCe8CEa565b84d4915e69e03d9C99d",
+      [CHAIN_IDs.GOERLI]: "0x40153DdFAd90C49dbE3F5c9F96f2a5B25ec67461",
+    },
+  },
+  BAL: {
+    name: "Balancer",
+    symbol: "BAL",
+    decimals: 18,
+    addresses: {
+      [CHAIN_IDs.MAINNET]: "0xba100000625a3754423978a60c9317c58a424e3D",
+      [CHAIN_IDs.OPTIMISM]: "0xFE8B128bA8C78aabC59d4c64cEE7fF28e9379921",
+      [CHAIN_IDs.POLYGON]: "0x9a71012B13CA4d3D0Cdc72A177DF3ef03b0E76A3",
+      [CHAIN_IDs.ARBITRUM]: "0x040d1EdC9569d4Bab2D15287Dc5A4F10F56a56B8",
+    },
+  },
+  BADGER: {
+    name: "Badger",
+    symbol: "BADGER",
+    decimals: 18,
+    addresses: {
+      [CHAIN_IDs.MAINNET]: "0x3472A5A71965499acd81997a54BBA8D852C6E53d",
+      [CHAIN_IDs.POLYGON]: "0x1FcbE5937B0cc2adf69772D228fA4205aCF4D9b2",
+      [CHAIN_IDs.ARBITRUM]: "0xBfa641051Ba0a0Ad1b0AcF549a89536A0D76472E",
+    },
+  },
+  BOBA: {
+    name: "Boba",
+    symbol: "BOBA",
+    decimals: 18,
+    addresses: {
+      [CHAIN_IDs.MAINNET]: "0x42bBFa2e77757C645eeaAd1655E0911a7553Efbc",
+      [CHAIN_IDs.BOBA]: "0xa18bF3994C0Cc6E3b63ac420308E5383f53120D7",
+    },
+  },
+  DAI: {
+    name: "Dai Stablecoin",
+    symbol: "DAI",
+    decimals: 18,
+    addresses: {
+      [CHAIN_IDs.MAINNET]: "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+      [CHAIN_IDs.OPTIMISM]: "0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1",
+      [CHAIN_IDs.POLYGON]: "0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063",
+      [CHAIN_IDs.BOBA]: "0xf74195Bb8a5cf652411867c5C2C5b8C2a402be35",
+      [CHAIN_IDs.ARBITRUM]: "0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1",
+      [CHAIN_IDs.GOERLI]: "0x5C221E77624690fff6dd741493D735a17716c26B",
+    },
+  },
+  ETH: {
```

Comment:
Should this be WETH instead of ETH?

Reply:
Or should this ETH key be deleted entirely?

Reply:
We also have `WETH` as a separate key [here](https://github.com/across-protocol/sdk-v2/pull/141/files/33852a83bc8bbfed0df37de883e603bdee8ece50#diff-8fa4b52909f895e8cda060d2035234e0a42ca2c7d3f8f8de1b35a056537bf199R142).

I am not sure if we can remove this safely because on the FE it seems that we sometimes use both 🤔 I would leave this for now and if we are sure that we don't need this on the FE then I can remove it.

Reply:
ok sgtm